package.json

```diff
@@ -59,7 +59,7 @@
     "@types/css-modules": "^1.0.2",
     "@types/jest": "29.4.0",
     "@types/node": "^18.15.3",
-    "@types/react": "18.0.28",
+    "@types/react": "^18.2.17",
```

Comment:
Updated this since I was getting the following warning messages during the build

> (!) Plugin typescript: @rollup/plugin-typescript TS2300: Duplicate identifier 'LibraryManagedAttributes'.
node_modules/@types/react-dom/node_modules/@types/react/ts5.0/index.d.ts: (3208:14)
...
...